manifests/vcsscm/git.pp

```diff
@@ -20,6 +22,26 @@
           ensure => $ensure,
         }
       }
+
+      if $bash_completion {
+        package { 'bash-completion':
+          ensure => $ensure,
+        }
+      }
+
+      if $bash_prompt {
+        vcsrepo { '/opt/bash-git-prompt/':
+          ensure   => present,
+          provider => git,
+          source   => 'https://github.com/magicmonty/bash-git-prompt.git',
+          depth    => 1,
+        }
+
+        file { '/etc/bash_completion.d/bash-git-prompt':
```

Comment:
If they are independent maybe this should be in /etc/profile.d?

Or maybe I'm reading this wrong and /etc/bash_completion.d has nothing to do with the bash_completion package...

Reply:
That directory is already there on recent Ubuntu systems. At least on 17.04 Zesty. Maybe we can find proof for it either in Bash documentation or on the Debian project, if there have been layout desicions or discussions.

Reply:
Ah! I get your confusion! You're right for profile to be the semantically more appropriate place. Though, the Git prompt feature is only for Bash (not all shells; see the project's README for details).

I really took that place, because placing a file in this folder allows you to keep the users default bashrc or profile untouched. I think, it's not really that bad as a hack, it's a clean integration with Bash.

Reply:
ok. That seems fine.